app/views/fsss_view.py

```diff
@@ -51,8 +52,9 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
     data_job_role = {
         'Job Role ID': current_job_role['uniqueRoleId'],
-        # badge number
+        'Badge Number': get_emp_info('idBadgeNo'),
```

Comment:
If the badge number was already present, it seems to me it should stay inside data_other.

Reply:
Moved back to where it should be